facebook-clang-plugins/clang/src/prepare_clang_src.sh

```diff
@@ -30,4 +38,8 @@ rm -rf "llvm-project-${LLVM_VER}.src" "llvm-project"
 tar xf "${LLVM_FILE}"
 mv "llvm-project-${LLVM_VER}.src" "llvm-project"
 
```

Comment:
Minor nit, but you lost the comment `# apply prebuild patch` when moving these lines.